identifiers/country-tw/township-level.csv

```diff
@@ -0,0 +1,70 @@
+id,name
+ocd-division/country:tw/special_municipality:tw-khh/district:constituency1,The first electoral district Kaohsiung City
```

Comment:
Could you update the naming of the districts to be consistent?

Reply:
done